superbench/cli/_help.py

```diff
@@ -61,6 +61,34 @@
           text: {cli_name} run --docker-image superbench/cuda:11.1 --host-file ./host.ini
 """.format(cli_name=CLI_NAME)
 
+helps['node'] = """
+    type: Group
+    short-summary: The command group of the node.
+"""
+
+helps['node info'] = """
+    type: command
+    short-summary: Get system info.
+    examples:
+        - name: get system info of the local node
+          text: {cli_name} node info
+""".format(cli_name=CLI_NAME)
+
+helps['result'] = """
+    type: Group
+    short-summary: The command group of the result processing.
+"""
+
+helps['result diagnosis'] = """
+    type: command
+    short-summary: Run data diagnosis.
```

Comment:
Filter the defective machines automatically from benchmarking results according to rules defined in rule file